frontend/apps/reader/modules/readerwikipedia.lua

```diff
@@ -38,7 +38,11 @@ function ReaderWikipedia:init()
 end
 
 function ReaderWikipedia:registerKeyEvents()
-    if Device:hasKeyboard() then
+    if Device:hasKeyboard() and Device.model == "Kindle3" then
+        -- Any user facing documentation should advertise this shortcut as Alt+W, as that is what
+        -- the user must input in order to use it.
+        self.key_events.ShowWikipediaLookup = { { "2" } }
+    elseif Device:hasKeyboard() then
```

Comment:
```lua
if Device:hasKeyboard()  then
    if Device.model == "Kindle3" then
    else
    end
end
```
or even, to not bug the rest of us reading this with kindlecrap:
```lua
if Device:hasKeyboard()  then
    self.key_events.ShowWikipediaLookup = { { "Alt", "W" }, { "Ctrl", "W" } }
    if Device.model == "Kindle3" then
        -- override previous ones on kindle3
        -- Any user facing documentation should advertise this shortcut as Alt+W, as that is what
        -- the user must input in order to use it.
        self.key_events.ShowWikipediaLookup = { { "2" } }
    end
end
```

Reply:
I thought you'd be happy we finally figured out, the curious case of the dog in the night time. https://github.com/koreader/koreader/pull/12078#issuecomment-2192719780

we now have it, without having to explicitly tell input text.

In any case `kindlecrap` is no longer, we are making kindle NT great again.